scripts/sign.sh

```diff
@@ -32,10 +32,9 @@ fi
 NAME="${1}"
 if [ -z "${2}" ]; then
   gpg --armor --output "${NAME}".asc --detach-sig "${NAME}"
-  gpg --print-md SHA512 "${NAME}" > "${NAME}".sha512
 else
   # The GPG key name to use
   GPG_LOCAL_USER="${2}"
   gpg --local-user "${GPG_LOCAL_USER}" --armor --output "${NAME}".asc --detach-sig "${NAME}"
-  gpg --local-user "${GPG_LOCAL_USER}" --print-md SHA512 "${NAME}" > "${NAME}".sha512
 fi
```

Comment:
This checks if a GPG user-id was specified to use `gpg` with `--local-user` for signing/decrypting. Generating SHA sums (`--print-md SHA512`) is not affected by specifying this `--local-user` option.

So it is the same as
```bash
if [ -z "${2}" ]; then
  gpg --armor --output "${NAME}".asc --detach-sig "${NAME}"
else
  # The GPG key name to use
  GPG_LOCAL_USER="${2}"
  gpg --local-user "${GPG_LOCAL_USER}" --armor --output "${NAME}".asc --detach-sig "${NAME}"
fi
gpg --print-md SHA512 "${NAME}" > "${NAME}".sha512
```

which I then replaced with `shasum`:

```bash
if [ -z "${2}" ]; then
  gpg --armor --output "${NAME}".asc --detach-sig "${NAME}"
else
  # The GPG key name to use
  GPG_LOCAL_USER="${2}"
  gpg --local-user "${GPG_LOCAL_USER}" --armor --output "${NAME}".asc --detach-sig "${NAME}"
fi
shasum -a 512 "${NAME}" > "${NAME}.sha512"
```